src/lib389/lib389/utils.py

```diff
@@ -1431,3 +1431,12 @@ def is_valid_hostname(hostname):
         hostname = hostname[:-1] # strip exactly one dot from the right, if present
     allowed = re.compile("(?!-)[A-Z\d-]{1,63}(?<!-)$", re.IGNORECASE)
     return all(allowed.match(x) for x in hostname.split("."))
+
+
+def snmp_present():
```

Comment:
I don't think this should be in lib389.
We have `get_rpm_version()` in `conftest.py`: https://github.com/389ds/389-ds-base/blob/a79433498709319814b7dae93c3dec9a0ed32c5c/dirsrvtests/conftest.py#L27-L34
It also won't fail on the systems where `rpm` is not present.